Python/executor_cases.c.h

```diff
@@ -4299,10 +4299,11 @@
             }
             STAT_INC(CALL, hit);
             _PyFrame_SetStackPointer(frame, stack_pointer);
-            res = PyStackRef_FromPyObjectSteal(PyObject_Str(arg_o));
+            PyObject *res_o = PyObject_Str(arg_o);
             stack_pointer = _PyFrame_GetStackPointer(frame);
             PyStackRef_CLOSE(arg);
-            if (PyStackRef_IsNull(res)) JUMP_TO_ERROR();
+            if (res_o == NULL) JUMP_TO_ERROR();
```

Comment:
Maybe curly braces should be added. For more to see PEP7.
```suggestion
            if (res_o == NULL) {
                JUMP_TO_ERROR();
            }
```
This suggestion applies to all changes.

Reply:
These are generated files; what you are suggesting makes more sense as a separate PR to the code generator.